server/src/main.cc

```diff
@@ -244,6 +245,11 @@ int mainRoutine(int argc, char *argv[])
 
 	// setup configuration database
 	ThreadLocalDBCache cache;
+
+	// Re execute unfinished action
+	ActionManager actionManager;
+	actionManager.reExecuteUnfinishedAction();
```

Comment:
memo: I've noticed that Hatohol server blocks this line when bunch of unfinished actions are remained.
But for now, this implementation is no problem to re-execute unfinished actions.